views/cellMove/partials/prisonerInformation.njk

```diff
@@ -44,4 +48,14 @@
       <p class="govuk-body" data-test="non-associations-message">0 in NOMIS. Check local processes.</p>
     {% endif %}
   </div>
-</div>
\ No newline at end of file
+</div>
+
+{{ govukInsetText({
+  html: '<p class="govuk-body"><a data-test="select-cswap-link" href="' + '/prisoner/' + offenderNo + '/cell-move/confirm-cell-move?cellId=C-SWAP"' + '  class="govuk-link">Create a space for another prisoner</a> -  this will leave ' + prisonerName + ' without a cell. You must move him into a cell as soon as possible today.',
```

Comment:
Not sure for now is this cross site script safe? I can never remember if it's automatic or now.